### PR TITLE
MCS-841:  Update code for database refactor

### PR DESCRIPTION
**Pull request by brianpippin**
This is the companion ticket for ingest MCS-841.  This code updates the UI to utilize the new structure where each eval has its own collection for results and scenes.

### CODE REVIEW

**Comment by brianpippin** [comment]
Note:  Old Saved Queries will not work with this until we complete the ticket for fixing those saved queries.  The ingest script to fix them will be very straight forward.

**Review by kdrumm — APPROVED** [review]
Code looks good as far as I understand it (which isn't that much)

**Review by ThomasSchellenbergNextCentury — APPROVED** [review]
Didn't notice anything weird

File: node-graphql/server.schema.js [hunk 1]
```diff
@@ -262,11 +274,16 @@ const mcsResolvers = {
             let performersArray = [];
             let metadataArray = [];
 
-            const statusObj = await mcsDB.db.collection('eval_status').find({"eval": args.eval})
+            console.log(args);
```
Comment (rartiss55) [inline node-graphql/server.schema.js:277]: Was this just console.log in here for debugging/should it get taken out?

**Review by deanwetherby — APPROVED** [review]

**Pull request merged**